crates/bounded_planet_core/src/networking/systems.rs

```diff
@@ -45,18 +63,16 @@ pub fn receive_net_events(
         match event {
             
             // A new connection has opened, allocate an entry in the hashmap
-            ReceiveEvent::Connected(id) => {
+            ReceiveEvent::Connected(id, ref packet_sender) => {
```

Comment:
Do you actually need this ref...? It's only used for cloning, I'd be really surprised if rust yells at you

Reply:
Using it here without a `ref` moves it out of the event. That's not valid because the event is later used (`net_events.send(event);`)